src/ucp/wireup/select.c

```diff
@@ -619,9 +619,14 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_add_lane_desc(
     for (lane_desc = select_ctx->lane_descs;
          lane_desc < select_ctx->lane_descs + select_ctx->num_lanes; ++lane_desc) {
         if ((lane_desc->rsc_index  == select_info->rsc_index) &&
-            (lane_desc->addr_index == select_info->addr_index) &&
-            (lane_desc->path_index == select_info->path_index))
-        {
+            (lane_desc->addr_index == select_info->addr_index)) {
+
+            /* Path index is only relevant for BW lanes */
+            if ((lane_desc->path_index != select_info->path_index) &&
```

Comment:
maybe have "any" value for select_info->path_index, which would be "equal" to any value.
we can set it before calling ucp_wireup_add_lane_desc, when we know that path_index is not important (such as am_lane for example)